requirements-cuda.txt

```diff
@@ -1,5 +1,6 @@
 # Common dependencies
 -r requirements-common.txt
+-r requirements-adag.txt
```

Comment:
actually, maybe let's remove this for now and ask people to manually download adag deps? We can update this in the future

Reply:
I moved this to requirements-test.txt , otherwise ADAG tests will fail.